android/build.gradle

```diff
@@ -22,5 +22,5 @@ def getModuleVersion() {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    api 'com.iterable:iterableapi:3.2.9'
+    api 'com.iterable:iterableapi:3.3.0-beta1'
```

Comment:
Ah Jay! Just realized. It needs to have GitHub link instead of com.iterable link for the beta versions